cookiecutter-django-app/{{cookiecutter.repo_name}}/test_settings.py

```diff
@@ -39,3 +42,20 @@ def root(*args):
 ROOT_URLCONF = '{{ cookiecutter.app_name }}.urls'
 
 SECRET_KEY = 'insecure-secret-key'
+
+MIDDLEWARE = (
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
```

Comment:
Why do we need these on the django-app cookie-cutter?

Reply:
These are required for admin checks to pass - running without them resulting in the following errors:

```
ERRORS:
?: (admin.E403) A 'django.template.backends.django.DjangoTemplates' instance must be configured in TEMPLATES in order to use the admin application.
?: (admin.E406) 'django.contrib.messages' must be in INSTALLED_APPS in order to use the admin application.
?: (admin.E408) 'django.contrib.auth.middleware.AuthenticationMiddleware' must be in MIDDLEWARE in order to use the admin application.
?: (admin.E409) 'django.contrib.messages.middleware.MessageMiddleware' must be in MIDDLEWARE in order to use the admin application.
?: (admin.E410) 'django.contrib.sessions.middleware.SessionMiddleware' must be in MIDDLEWARE in order to use the admin application.
```